Orange/widgets/unsupervised/owdistances.py

```diff
@@ -160,6 +161,8 @@ def checks(metric, data):
         except MemoryError:
             self.Error.distances_memory_error()
             return
+
+        met = np.nan_to_num(met)
```

Comment:
I was about to write that while I totally agree with @janezd, most solutions for inf distances will probably be equivalent to substitution with large numbers. In which case maybe we should consider if we want every widget to duplicate this substitution.
But there is always the possibility that somewhere inf numbers would be illegal (and we would want a warning). And even worse - this substitution will probably not allow downstream methods to ignore the problem, since you still cannot multiply the new "large values", or add two of them etc. which could happen in average linkage and so on.